src/DynamoPackages/PackageLoader.cs

```diff
@@ -828,6 +828,8 @@ internal void DoCachedPackageUninstalls(IPreferences preferences)
                     Directory.Delete(pkgNameDirTup, true);
                     pkgDirsRemoved.Add(pkgNameDirTup);
                     Log(String.Format("Successfully uninstalled package from \"{0}\"", pkgNameDirTup));
+                    string package = System.IO.Path.GetFileNameWithoutExtension(pkgNameDirTup);
+                    Analytics.TrackEvent(Actions.Delete, Categories.PackageManagerOperations, $"Delete the Package {package} scheduled");
```

Comment:
I appologize, this has already been covered in Dynamo code in `PackageManagerExtension.cs`, we can remove this

Reply:
Done